docs/architecture/porting-existing-aspnet-apps/migrate-aspnet-core-2-1.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Migrate to ASP.NET Core 2.1
+description: As the last .NET Core version to support .NET Framework runtime targeting, does migrating to .NET Core 2.1 make sense as an intermediate step in some app migration plans?
+author: ardalis
+ms.date: 11/13/2020
+---
+
+# Migrate to ASP.NET Core 2.1
+
+ASP.NET Core 2.1 is an interesting release because it's the only currently supported .NET Core release to support both .NET Core and .NET Framework runtimes. As such, it may offer an easier upgrade path for some apps when compared to upgrading all parts of the app to .NET Core at once. As an LTS release, support for .NET Core 2.1 will continue through August 2021.
```

Comment:
This block is confusing - it might make sense to more clearly distinguish between "ASP.NET Core 2.1" and ".NET Core 2.1".

For example, "ASP.NET Core 2.1" on .NET Framework will be supported as long as it's underlying .NET Framework is supported, so the part about LTS is confusing.

Reply:
Clarified.